drawer.go

```diff
@@ -44,12 +50,30 @@ func (d *Drawer) DrawLabelAsPng(label elements.LabelInfo, output io.Writer, opti
 	gCtx.Clear()
 
 	for _, element := range label.Elements {
+		reversePrint := false
+
+		if el, ok := element.(reversePrintable); ok {
+			reversePrint = el.IsReversePrint()
+		}
+
+		gCtx2 := gCtx
+		if reversePrint {
+			gCtx2 = gg.NewContext(int(imageWidth), int(imageHeight))
+			gCtx2.SetRGBA(0, 0, 0, 0)
+			gCtx2.Clear()
+		}
+
 		for _, drawer := range d.elementDrawers {
-			err := drawer.Draw(gCtx, element, options)
+			err := drawer.Draw(gCtx2, element, options)
 			if err != nil {
 				return fmt.Errorf("failed to draw zpl element: %w", err)
 			}
 		}
+
+		if reversePrint {
+			xorImg := images.NewXor(gCtx2.Image(), gCtx.Image())
+			gCtx.DrawImage(xorImg, 0, 0)
```

Comment:
You're doing XOR and the output is like that text, right?
<img width="198" alt="Screenshot 2024-03-21 at 16 42 46" src="https://github.com/ingridhq/zebrash/assets/83748753/ee72c8cb-6a1d-4e80-a1b1-571aeacfbf50">
Shouldn't it be called "inversion" instead or "reverse"?

Reply:
Yeah, renamed it to ReversePrint

Reply:
As for why it is called Reverse and not Inverse, it is named like this in ZPL manuals/specifications
![Screenshot from 2024-03-21 18-26-57](https://github.com/ingridhq/zebrash/assets/78364711/6fb700cd-8e77-4f99-989c-65d421e02299)